### PR TITLE
Use masternode list to generate hardcoded seeds

### PR DESCRIPTION
**Pull request by codablock**

### CODE REVIEW

**Comment by codablock** [comment]
Someone should additionally verify that they get the same list of seeds when running the scripts as described in the modified README.md.

Also, we should think about the need of the asn filtering in Dash. I assume this filtering was added to make it harder for sybil or other malicious nodes to somehow trick the DNS and thus get into the hardcoded seed list. But as we use the MN list for this, which is already protected from such things, we probably don't need the filtering at all.

**Comment by schinzelh** [comment]
@codablock I think we should stick to ASN filtering, that way we better distribute between datacenters. We know that a large portion of the masternodes are hosted on Cloud providers, e.g. Amazon, Digital Ocean, Vultr. So filtering mitigates the risk on being to reliant on them.

File: contrib/seeds/README.md [hunk 1]
```diff
@@ -3,13 +3,12 @@
 Utility to generate the seeds.txt list that is compiled into the client
 (see [src/chainparamsseeds.h](/src/chainparamsseeds.h) and other utilities in [contrib/seeds](/contrib/seeds)).
 
-Be sure to update `PATTERN_AGENT` in `makeseeds.py` to include the current version,
-and remove old versions as necessary.
+Be sure to update `MIN_PROTOCOL_VERSION` in `makeseeds.py` to include the current version.
 
 The seeds compiled into the release are created from sipa's DNS seed data, like this:
```
Comment (UdjinM6) [inline contrib/seeds/README.md]: This is not accurate

**Comment by codablock** [comment]
@schinzelh True, was thinking about the hosting situation as well. But in regard to that, the effect of better distribution is immediately lost when the node connects to the first node, as it will immediately receive a list of new addresses where no ASN filtering was applied.

At the same time, the network code applies some logic to distribute between network "groups", which in the end results in the same: distribution across different datacenters and cloud providers.

Also, as smaller the list of seeds is, the more interesting these MNs get for DoS attacks. An attacker could try to kill one entry at a time and reset their "activeseconds" counter, resulting in these nodes being chosen last for the ASN filtering. At the same time he'd keep his own nodes alive, making them more and more likely to get into the list.

With more seed entries, this gets less and less effective.

**Comment by UdjinM6** [comment]
tested ACK

**Comment by UdjinM6** [comment]
re-ACK c85174b8464ea682b5af7cc5a85603b25c3976cb

**Pull request merged**